app/errors/max_allow_targets_error.rb

```diff
@@ -0,0 +1,2 @@
+class MaxAllowTargetsError < StandardError
+end
```

Comment:
this could be in `module Target` and have it in a properly folder like `app/errors/target/max_allow_targets_error.rb` maybe